server/app/com/xsn/explorer/models/request/TposContractsEncodeRequest.scala

```diff
@@ -0,0 +1,20 @@
+package com.xsn.explorer.models.request
+
+import play.api.libs.functional.syntax._
+import play.api.libs.json._
+
+case class TposContractsEncodeRequest(
+    tposAddress: String,
+    merchantAddress: String,
+    commission: Int,
+    signature: String
+)
+
+object TposContractsEncodeRequest {
+  implicit val reads: Reads[TposContractsEncodeRequest] = (
```

Comment:
I believe this can even be auto-generated by = `Json.reads[TposContractsEncodeRequest]`